gravitee-plugin-core/pom.xml

```diff
@@ -43,10 +43,22 @@
             <artifactId>gravitee-common</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.reactivex.rxjava3</groupId>
```

Comment:
need to remove that updating to prent pom @ 21.0.0

Reply:
21.0.1 ! You also need to update the orb vers to 4.0.0 in circle.yml